server/chat.ts

```diff
@@ -1619,6 +1622,15 @@ export const Chat = new class {
 		return `${num}${space}${num > 1 ? pluralSuffix : singular}`;
 	}
 
+	/**
+	 * Takes the name of a command and gets the base command, if there is one.
+	 * @param cmd string
+	 */
+	baseCommand(cmd: string) {
+		if (typeof this.commands[cmd] === 'string') return this.commands[cmd] as string;
```

Comment:
```suggestion
		if (typeof this.commands[cmd] === 'string') return this.commands[cmd];
```
This should be implied by the if statement immediately beforehand

Reply:
Not implied, I presume since the typing of chat commands allows for it to be several things, so although the actual code ensures it will always be a string, typescript doesn't acknowledge that `Chat.commands[alias]` will be a string.